decidim-meetings/app/commands/decidim/meetings/close_meeting.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    # This command is executed when the user closes a Meeting from the admin
+    # panel.
+    class CloseMeeting < Rectify::Command
+      # Initializes a CloseMeeting Command.
+      #
+      # form - The form from which to get the data.
+      # meeting - The current instance of the page to be closed.
+      def initialize(form, meeting)
+        @form = form
+        @meeting = meeting
+      end
+
+      # Closes the meeting if valid.
+      #
+      # Broadcasts :ok if successful, :invalid otherwise.
+      def call
+        return broadcast(:invalid) if form.invalid?
+
+        transaction do
+          close_meeting
+          link_proposals
+        end
+
+        broadcast(:ok)
+      end
+
+      private
+
+      attr_reader :form, :meeting
+
+      def close_meeting
+        parsed_closing_report = Decidim::ContentProcessor.parse_with_processor(:hashtag, form.closing_report, current_organization: form.current_organization).rewrite
```

Comment:
If you use `parse` here it will process the text with all the processors, not just the hashtag, see https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/content_processor.rb#L53

```suggestion
        parsed_closing_report = Decidim::ContentProcessor.parse(form.closing_report, current_organization: form.current_organization).rewrite
```